apiserver/pkg/modelservice/modelservice.go

```diff
@@ -162,20 +165,23 @@ func UpdateModelService(ctx context.Context, c dynamic.Interface, input generate
 }
 
 func DeleteModelService(ctx context.Context, c dynamic.Interface, input *generated.DeleteCommonInput) (*string, error) {
-	_, err := embedder.DeleteEmbedders(ctx, c, input)
-	if err != nil {
-		return nil, err
+	var errText string
+	_, err1 := embedder.DeleteEmbedders(ctx, c, input)
+	if err1 != nil {
+		errText += "embedder: " + err1.Error()
 	}
-	_, err = llm.DeleteLLMs(ctx, c, input)
-	if err != nil {
-		return nil, err
+	_, err2 := llm.DeleteLLMs(ctx, c, input)
+	if err2 != nil {
+		errText += " llm:" + err2.Error()
+	}
+	if errText != "" {
+		return nil, errors.New("error occurred during deleting: " + errText)
```

Comment:
Use `"github.com/pkg/errors"`  instead of  golang `errors`

And use `errors.Errorf()`